README.md

```diff
@@ -52,14 +48,13 @@ You can use a tool like [nvm](https://github.com/nvm-sh/nvm) to get the version
 
 ## 🚀 (Frontend) Setup
 
-1. Clone the project with `git clone git@github.com:Code-For-Chicago/Voma-frontend.git`
```

Comment:
Let's keep this step, there have been a few people who've forked instead of cloned so it's good to call out.